src/buildLocalPackages.cmd

```diff
@@ -2,11 +2,10 @@
 setlocal enabledelayedexpansion
 
 set MSBUILDARGS=-p:PublishRepositoryUrl=true -p:GeneratePackages=true -p:IncludeSymbols=true -p:SymbolPackageFormat=snupkg -p:InternalBuild=true -p:Configuration=Debug -p:Platform="Any CPU"
+set MSBUILD=C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\amd64\MSBuild.exe
 
-for /f "usebackq tokens=*" %%i in (`"%programfiles(x86)%\Microsoft Visual Studio\Installer\vswhere" -latest -requires Microsoft.Component.MSBuild -find MSBuild\**\Bin\MSBuild.exe`) do (
```

Comment:
What is this? (did you intend to check in)

Reply:
Yes, this script doesn't work anymore so I fixed it
It used to search for latest VS version but this approach gives you VS2019 and not VS2022